src/main/java/seedu/address/logic/commands/RestockIngredientCommand.java

```diff
@@ -3,7 +3,8 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_INGREDIENT_QUANTITY;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_ITEMS;
+import static seedu.address.model.IngredientModel.PREDICATE_SHOW_ALL_INGREDIENTS;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_INGREDIENTS;
```

Comment:
why two separate imports for the same thing??

Reply:
fixed